apps/api/src/app/admin/admin.controller.ts

```diff
@@ -221,7 +221,7 @@ export class AdminController {
     @Param('dataSource') dataSource: DataSource,
     @Param('dateString') dateString: string,
     @Param('symbol') symbol: string
-  ): Promise<MarketData> {
+  ): Promise<MarketData | number> {
```

Comment:
Please keep `MarketData` only

Reply:
Then how to return `marketPrice` from `data-gathering.service` which is of type `number` ?

Reply:
See my comment [here](https://github.com/ghostfolio/ghostfolio/pull/2546#discussion_r1375308167).